src-backbone/app/js/models/session.js

```diff
@@ -96,6 +97,46 @@ let SessionModel = Backbone.Model.extend({
         });
     },
 
+    updateInformation: function(formData) {
+        let self = this;
+        let json = {};
+        formData.forEach(function(item) {
+            if (item.value !== "") {
```

Comment:
The form data itself will always give us an empty string for a field if it hasn't been filled in.